content/rc/concepts/clustering.md

```diff
@@ -72,10 +72,10 @@ are supported with the following limitations:
     and pipelining are supported with Redis Cloud cluster
     like if it were a non-cluster DB.
 
-## Changing the sharding policy
+## Changing the hashing policy
 
 The clustering configuration of a Redis Cloud instance can be
```

Comment:
I think we can change the line (76) - "The clustering configuration of a Redis cloud instance.." 
To: 
"The hashing policy of a Redis Cloud instance can be changed" or "is configurable."

Then it will fit perfectly to the rest of the line that you modified - "However, hashing policy changes..."

Reply:
+1